src/components/tasks/TasksContent.tsx

```diff
@@ -12,14 +12,34 @@ import { TabSection } from './TabSection';
 import TaskList from './TaskList/TaskList';
 import updateCardContent from '@/helperFunctions/updateCardContent';
 import { useEditMode } from '@/hooks/useEditMode';
+import { useRouter } from 'next/dist/client/router';
 
 export const TasksContent = () => {
-    const { data: tasks = [], isError, isLoading } = useGetAllTasksQuery();
+    const router = useRouter();
+    const { dev } = router.query;
     const { isEditMode } = useEditMode();
     const isUserAuthorized = useContext(isUserAuthorizedContext);
     const isEditable = isUserAuthorized && isEditMode;
     const [activeTab, setActiveTab] = useState(Tab.IN_PROGRESS);
 
+    const {
+        data: tasks = [],
+        isError,
+        isLoading,
+    } = useGetAllTasksQuery({
+        dev: dev as unknown as boolean,
+        status: activeTab,
+    });
+
+    const {
+        data: filteredTaskResponse,
+        isLoading: isTaskLoading,
+        isError: isTaskError,
+    } = useGetAllTasksQuery({
+        dev: dev as unknown as boolean,
```

Comment:
I don't understand the need to do `dev as unknown as boolean`. Can you please explain?

Reply:
Done